CMakeLists.txt

```diff
@@ -132,14 +132,15 @@ endif()
 # Python module
 if(TRITON_BUILD_PYTHON_MODULE)
     message(STATUS "Adding Python module")
+    set(PYTHON_SRC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/python/src)
+    include_directories("." ${PYTHON_SRC_PATH})
     if (PYTHON_INCLUDE_DIRS)
-      set(PYTHON_SRC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/python/src)
-      include_directories("." ${PYTHON_SRC_PATH} ${PYTHON_INCLUDE_DIRS})
+      include_directories(${PYTHON_INCLUDE_DIRS})
       link_directories(${PYTHON_LINK_DIRS})
+      link_libraries(${PYTHON_LIBRARIES})
```

Comment:
What happens if we comment this line, do we get like a linker warning that then becomes an error with the new options?

Reply:
The link fails, but i don't believe its a "warning", i believe its failing with missing references. Let me see if i can repro without the warning changes.